### PR TITLE
feat(config): update MP20Z config

### PR DESCRIPTION
**Pull request by emichael**
This adds all of the configuration options described by the manufacturer. It also updates the manual to a more complete one from the manufacturer's website.

If you don't want to have a manual link that's not the z-wave alliance, I can back that line out.

Also, this was done entirely in-browser and without testing. I don't have a decent dev environment setup. Should be straightforward enough, though. I just looked at the manual I linked below and started from the MP21Z paramInformation. The one thing I was uncertain about was why the MP21Z has a maxValue of 3 for param 1. Is the maxValue supposed to be exclusive? If so, all of those should be bumped up by one.

### CODE REVIEW

**Comment by emichael** [comment]
Can't see any more details, so I'm not sure what's causing the "PR title / commit messages" failure. Seems pretty semantic to me!

**Comment by glyph-se** [comment]
> 
> 
> Can't see any more details, so I'm not sure what's causing the "PR title / commit messages" failure. Seems pretty semantic to me!

Try changing the commit message to the same as the title on the PR

**Comment by AlCalzone** [comment]
Yeah that's it. Unfortunately, GitHub has the stupid habit of ignoring the PR title in its merge commit messages if there is only one commit.

File: packages/config/config/devices/0x0312/mp20z.json [hunk 1]
```diff
@@ -14,10 +14,118 @@
...
+					"label": "Output On",
+					"value": 1
+				},
+				{
+					"label": "Restore Outlets to Previous State",
+					"value": 2
+				}
+			]
+		}
+	},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0312/mp20z.json:104]: We have [several templates](https://github.com/zwave-js/node-zwave-js/blob/086ac3e67a36f74e4157c46c3511529d22700520/packages/config/config/devices/templates/master_template.json#L256) for these, please choose the matching one and [use it](https://zwave-js.github.io/node-zwave-js/#/config-files/using-templates?id=using-templates).

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0312/mp20z.json (hunk 1 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0312/mp20z.json:104]: ...that comment was meant to go on the "State after power loss" parameter.

Comment (emichael) [reply packages/config/config/devices/0x0312/mp20z.json:104]: Done!

**Comment by emichael** [comment]
Aggg, sorry about the import path. Really should have set something up to test it locally before pushing. (Side note: there probably should be a way to refer to the templates directory independent of the current directory, no? I get that the folder structure of the project is pretty mature at this point, but it still seems unnecessarily brittle?)

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**